bench/bench.py

```diff
@@ -147,11 +147,9 @@ def run():
             for cmd in [
                     'sudo -S pkill -f nsqd',
                     'sudo -S rm -f /mnt/nsq/*.dat',
-                    'GOMAXPROCS=32 ./go/src/github.com/nsqio/nsq/apps/nsqd/nsqd \
-                        --data-path=/mnt/nsq --mem-queue-size=10000000 --max-rdy-count=%s' % (
-                        tornado.options.options.rdy
-                        )]:
-                nsqd_chans.append((ssh_client, ssh_cmd_async(ssh_client, cmd)))
+					'echo starting nsq &>> nsq.log',
+                    'GOMAXPROCS=32 screen -mdS "nsq" ./go/src/github.com/nsqio/nsq/build/nsqd --data-path=/mnt/nsq --mem-queue-size=10000000 --max-rdy-count=%s &>> nsq.log' % (tornado.options.options.rdy)]:
```

Comment:
While we have no explicit formatting rules for python helper scripts in this repo, this line is a bit too long. Please wrap somewhere around 100 chars.

Here's a trick: in python, if there are two string literals next to each other, they are concatenated by the python parsing step. Something like this should work:

```python
            for cmd in [
                'sudo -S pkill -f nsqd',
                'sudo -S rm -f /mnt/nsq/*.dat',
                'echo starting nsq &>> nsq.log',
                'GOMAXPROCS=32 screen -mdS "nsq" ./go/src/github.com/nsqio/nsq/build/nsqd'
                ' --data-path=/mnt/nsq --mem-queue-size=10000000'
                ' --max-rdy-count=%s &>> nsq.log' % tornado.options.options.rdy,
            ]:
```

(admittedly a bit tricky, since a missing or present comma makes a big difference, silently)